src/main/webapp/app/tasks/task-list/task-list.component.html

```diff
@@ -1,9 +1,24 @@
-<mat-list data-cy="task-list">
-  <mat-list-item *ngFor="let task of tasks" class="mat-elevation-z1">
-    <mat-icon mat-list-icon>assignment</mat-icon>
+<div class="task-list-menu mat-elevation-z1">
+
+  <button mat-icon-button aria-label="Delete all done tasks" color="primary" matTooltip="Delete all done tasks">
+    <mat-icon color="warn" aria-label="Delete all done tasks" (click)="deleteAllDone(task)">remove_done</mat-icon>
+  </button>
+</div>
+
+
+<mat-list cdkDropList (cdkDropListDropped)="drop($event)" data-cy="task-list">
```

Comment:
nice that you implemented the drag & drop but what's the functional use? a user cannot use it to mark something as done

Reply:
I was aware of this. But I see no point in drag&drop later. Because I already had a checkbox. I was a bit confused of requirements. It mentions two tables but then also mentions of moving up and down a task. In any case, I should have removed drag & drop code (maybe in the next commit :)

Reply:
I lated decided that I do not need a drag&drop behavior. The code is there for an example :) (Joking, I should have removed that code)